docs/application/native/guides/notification/notification-ex.md

```diff
@@ -0,0 +1,528 @@
+   char *shared_path = app_get_shared_resource_path();
+   snprintf(image_path, BUFLEN, "%stutorial_native_api_application.png", shared_path);
+   free(shared_path);
+   ```
+
+<a name="create"></a>
+## Creating a notification-ex Items
+
+Each element that makes up a notification is an item. Text, Image, Time, etc.
+An item can be a notification by itself, and each item can be combined to create a group notification.
+
+- Text Item:
+
+  This item can set text and hyper link. It can be used in various ways such as title and contents.
+  ```
+  ret = noti_ex_item_text_create(&text_item, "text_id", "text", "hyperlink");
+  if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+  ```
+
+- Image Item:
+
+  This item can set an image or icon. Contains path information of the image.
+  ```
+  ret = noti_ex_item_image_create(&image_item, "image_id", image_path);
+  if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+  ```
+
+- Button Item:
+
+  This item can add a button.
+  ```
+  ret = noti_ex_item_button_create(&button_item, "button_id", "button_title");
+  if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+  ```
+
+- Chat Message Item:
+
+  This item can create a notification style of messaging. It is a combination of text item, image item and time item.
+  An application can construct multiple chat message items and display them in bubble form.
+  ```
+  ret = noti_ex_item_chat_message_create(&chatmessage_item, "message_id", name_handle,
+	text_handle, image_handle, time_handle, NOTI_EX_ITEM_CHAT_MESSAGE_TYPE_USER);
+  if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+  ```
+
+- CheckBox Item:
+
+  This item can provide a user selectable check box. An item that can be toggled. The state of check and uncheck can be determined in advance.
+  ```
+  ret = noti_ex_item_checkbox_create(&checkbox_item, "checkbox_id", "checkbox_title", false);
+  if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+  ```
+
+- Input Selector Item:
+
+  This item allows you to predefine the answer the user can choose from the noti for which a reply is requested.
+  ```
+  noti_ex_item_h selector_item = NULL;
+  ret = noti_ex_item_input_selector_create(&selector_item, "selector_id");
+  if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+
+  const char *contents[] = {"What's up?", "On my way", "Nice"};
+  ret = noti_ex_item_input_selector_set_contents(selector_item, contents, 3);
+  if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+  ```
+
+- Group Item:
+
+  This item can make a group from multiple items.
+  ```
+  noti_ex_item_h group_item = NULL;
+  noti_ex_item_h child_item = NULL;
+
+  ret = noti_ex_item_button_create(&child_item, "button_id", "button_title");
+
+  ret = noti_ex_item_group_create(&group_item, "group_id");
+  if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+
+  ret = noti_ex_item_group_add_child(group_item, child_item);
+  if (ret != NOTI_EX_ERROR_NONE)
+    /* Error handling */
+```
```

Comment:
Please use same indentation with the starting code block line in 147.
We need 2 spaces in front of \`\`\`.

Reply:
use 4 spaces for whole file.